ynput/ayon-frontend
# Cloud: show trial banner

## Description of changes 

[Applies only to Ynput Cloud instances]
- Creates a free trial banner when the instance has a subscription with `status: trialing`.
- Shows a free trial over screen when the trial has finished.
- Show or hide auth providers using a query param `provider`. Useful for redirects from Ynput Cloud.
- Adds support for custormerly chat when trialing.



## Testing

1. Connect instance to Ynput Cloud
2. Create a new test subscription in supabase and set to `status: trialing` with `end_trial` date in the future.
3. See if trial banner appears on bottom of AYON
4. Change the trial end date to less than one day before now, see if the banner turns orange.
5. Change the trial end date to less than one hour before now, see if the banner turns red.
6. Change the trial end date to before now and see if trial over screen is shown.
